index.js

```diff
@@ -136,6 +139,7 @@ defineMacro('CC_JSB', defined('jsb'));
 defineMacro('CC_BUILD', false);
 defineMacro('CC_WECHATGAME', false);
 defineMacro('CC_QQPLAY', false);
+defineMacro('CC_FB_PLAYABLE_ADS', defined('FbPlayableAd'));
```

Comment:
I think it's not necessary to add a macro here. Just use cc.sys.platform is more unified